activerecord/test/cases/adapters/postgresql/prepared_statements_test.rb

```diff
@@ -0,0 +1,17 @@
+require "cases/helper"
+require 'models/developer'
+
+class PreparedStatementsTest < ActiveRecord::PostgreSQLTestCase
+  fixtures :developers
+
+  def setup
+    Developer.connection_config[:prepared_statements] = false
```

Comment:
This pollutes the config for subsequent tests; we need a teardown that puts it back how it was